package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Hi, this is looking great, but would you mind removing the package-lock.json from your PR, as I think it's just a version tweak? https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git

Reply:
Hi, thanks for the feedback. I'll fix it soon.